powerspikegg/frontend/scripts/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+ROOT=$(PWD)
+OUT_DIR="$ROOT/$1"
+GENDIR="$ROOT/$2"
+
+# Position the terminal into the frontend directory
+cd "powerspikegg/frontend"
+
+# Compile client
+export USERPROFILE="$ROOT/$GENDIR/.appdata"
+export HOME=$USERPROFILE
+(./node_modules/.bin/ng build --output-path "$OUT_DIR") || true
```

Comment:
Bazel sandboxing conflict with angular-cli environment, however the build still work but the return code might differ depending on the environment and we have to ignore it for Bazel to continue building. (to be investigated later)